pkgs/development/python-modules/opentelemetry-exporter-prometheus/default.nix

```
@@ -13,7 +14,7 @@
 buildPythonPackage {
   inherit (opentelemetry-api) src;
   pname = "opentelemetry-exporter-prometheus";
-  version = "0.44b0";
+  version = opentelemetry-instrumentation.version;
```

```suggestion
  version = opentelemetry-api.version;
```

Surely this should use the same version as the package which provides `src`.

Same everywhere else.

---

Oh I see now that you're saying these are different for some reason.

Can you instead add a comment explaining this surprising effect to each location so that someone like me doesn't come and "fix" it?

---

Fair enough. Thanks for your suggestion.

---

Noting that with this change, upgrading these modules might require an upgrade of `upentelemetry-instrumentation` so that the versions in the source match. I don't love that, but perhaps it's ok as long as the comment makes that nuance clear.

---

Thanks for the insight!
I've already set up the updateScript to update both at the same time. ref. https://github.com/NixOS/nixpkgs/commit/32da45d943575e585c4a03b1d2b9039745c5c545